### ISSUE TITLE
Latest version is not on npm

### ISSUE THREAD

**Issue by kamilwaheed** [senecajs/seneca-mongo-store#32]
I see that the mongodb dependency was updated a few months ago to ~2.1.2 but that does not reflect in the package from npm.

Please bump up the version and push to npm.

**Comment by mcdonnelldean** [comment]
This will be happening soon. There are some issues we need to fix and then we will push. apologies for the delay.

**Comment by luhtonen** [comment]
How soon this will happen? Current version producing error with latest Node.js:
```
016-08-05T07:36:15.059Z m4mnh83ah93n/1470382575028/67602/- INFO	hello	Seneca/2.1.0/m4mnh83ah93n/1470382575028/67602/-	
{ Error: Cannot find module '../build/Release/bson'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/.../product-management/node_modules/bson/ext/index.js:15:10)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3) code: 'MODULE_NOT_FOUND' }
js-bson: Failed to load c++ bson extension, using pure JS version```
Node.js version: v6.3.1
npm version: 3.10.3

**Comment by mihaidma** [comment]
@kamilwaheed @luhtonen published a the 1.0.0 version of mongo store.
Please let us know if you encounter issues
